akka-stream-tests/src/test/scala/akka/stream/scaladsl/SetupSpec.scala

```diff
@@ -11,9 +11,7 @@ import akka.stream.testkit.StreamSpec
 
 @nowarn("msg=deprecated")
 class SetupSpec extends StreamSpec {
-
-  import system.dispatcher
```

Comment:
@octonato @johanandren The reason is, I think the dispatcher was trying to invite itself to the party ;)

Reply:
oh, now I see your comment. Yeah, that's what I discovered.